browser-tests/BirthdayForm.ts

```diff
@@ -0,0 +1,34 @@
+import { loginSelector } from './pages/loginSelector';
+import { testURL } from './utils/settings';
+
+fixture('Open birth date form').page(testURL());
+
+test('Test invalid dates', async t => {
```

Comment:
Testing with this kind of detail ("wrong month dates don't work", "leap day won't work" and "wrong month won't work") suggests a more unit testing like mindset. I'm not sure if they are good test candidates for `e2e` tests. For `e2e` tests I'd consider more user facing test cases, for instance "X-Y year olds can register" and ">Z year olds can't register". You already seem to have these cases covered.

Also, invalid date behaviour has been tested with unit(/integration) tests on the `DateInput` level. In that sense the behaviour is covered. Although I only tested a single non-existent date, because I considered that to cover enough ground. I would have placed these test cases as unit/integration tests in the `DateInput` files.

Reply:
I'll remove these.